src/ephemeris.c

```diff
@@ -21,6 +21,8 @@
 #include <libswiftnav/constants.h>
 #include <libswiftnav/ephemeris.h>
 
+#define GLO_NUM_STEPS 30 /* For Runge-Kutta algorithm */
```

Comment:
this should be 
/* maximum step length in seconds for Runge-Kutta aglorithm */
#define GLO_MAX_STEP_LENGTH 30